pkg/server/httpserver.go

```diff
@@ -120,6 +120,14 @@ func FormatAndReturnTextResponse(res http.ResponseWriter, data string) {
 	return
 }
 
+// FormatAndReturnOctetResponse formats the given data as plaintext and returns the response
```

Comment:
nit: "formats the given data into an *octet stream* and returns the response"

Reply:
Done